### PR TITLE
implement VxpLabel, update docs

### PR DESCRIPTION
**Pull request by osmanertem**

### CODE REVIEW

File: docs/components/vxp-label.md [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
+This is an overview of the most common usage of `<VxpLabel>`.
 
-`Label` is a UI component that displays read-only text.
+`VxpLabel` is a UI component that displays read-only text.
 
 ## Props
 
 | Name     | Type    | Default | Web | Mobile |
 | -------- | ------- | ------- | --- | ------ |
 | text     | String  |         | ✔   | ✔      |
```
Comment (ahmetceylan) [inline docs/components/vxp-label.md]: can you please add description column, it has been probably forgotten

Comment (osmanertem) [reply docs/components/vxp-label.md]: description column is added

File: docs/components/vxp-label.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+sidebarDepth: 0
+---
+
+# VxpLabel
+## Live Demo
```
Comment (volkanakbayir) [inline docs/components/vxp-label.md]: should there be a little explanation what this component does?
There are explanations in other components here

File: docs/components/vxp-label.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+sidebarDepth: 0
+---
+
+# VxpLabel
```
Comment (volkanakbayir) [inline docs/components/vxp-label.md]: should there be a little explanation what this component does?
There are explanations in other components here

File: tests/unit/components/VxpLabel.spec.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import { mount } from '@vue/test-utils';
+import VxpLabel from '../../../src/components/VxpLabel.vue';
+import Label from '../../../src/core/components/Label/Label';
+
+describe('VxpLabel.vue', () => {
+  const wrapper = mount(VxpLabel);
+  it('VxpLabel includes a label', () => {
+    expect(wrapper.find(Label).exists()).to.equal(true);
+  });
+});
```
Comment (volkanakbayir) [inline tests/unit/components/VxpLabel.spec.js]: here needs to have a test for given text is properly passed to inner Label component.

File: docs/components/vxp-label.md (hunk 3 above)
Comment (zhunor) [reply docs/components/vxp-label.md]: @volkanakbayir I think explanation is at line17 it might be moved up here

**Pull request merged**